homeassistant/components/zwave_js/const.py

```diff
@@ -133,4 +133,5 @@
     vol.Coerce(float),
     BITMASK_SCHEMA,
     cv.string,
+    dict
```

Comment:
`black` needs a trailing comma
```suggestion
    dict,
```